sequencer/README.md

```diff
@@ -78,6 +78,8 @@ To run a node, you need to have a valid committee file and valid parameters (`ma
 
 Consensus will start when nodes can communicate with each other, which means you need to run all nodes. This in turn means all nodes need to share the same config and no nodes can be added to the network afteward.
 
+In order to change the execution engine from `cairo_native` to `cairovm` you will need to modify the function `local` in `fabfile.py`, change the `execution` parameter to `cairovm`.
```

Comment:
While this is the case if you use `fab local` (or `make bench`) from this directory, it might also be worth adding in the top level readme so that it is presented as an option for the docker solution (notice that for this you need to modify [this](https://github.com/lambdaclass/starknet_stack/blob/main/sequencer/config/parameters.json) committed file instead of the fabfile). What do you think?

Reply:
Great idea!